compiler/testData/diagnostics/tests/inference/constraints/wrongApproximationWithDefNotNullTypesAndDelegates.kt

```diff
@@ -4,7 +4,7 @@
 import kotlin.reflect.KProperty
 
 class Foo {
-    var test: String by refreshOnUpdate("str")
+    var test: String by <!INAPPLICABLE_CANDIDATE!>refreshOnUpdate<!>("str")
```

Comment:
This is due to https://youtrack.jetbrains.com/issue/KT-45344

Reply:
You should remove `// FIR_IDENTICAL` and update the generated `.fir.kt` instead.